src/sql/platform/theme/common/colorRegistry.ts

```diff
@@ -66,3 +66,5 @@ export const codeEditorLineNumber = registerColor('notebook.codeEditorLineNumber
 export const codeEditorToolbarIcon = registerColor('notebook.codeEditorToolbarIcon', { light: '#999999', dark: '#A19F9D', hc: '#FFFFFF' }, nls.localize('notebook.codeEditorToolbarIcon', "Notebook: Code editor toolbar icons"));
 export const codeEditorToolbarBackground = registerColor('notebook.codeEditorToolbarBackground', { light: '#EEEEEE', dark: '#333333', hc: '#000000' }, nls.localize('notebook.codeEditorToolbarBackground', "Notebook: Code editor toolbar background"));
 export const codeEditorToolbarBorder = registerColor('notebook.codeEditorToolbarBorder', { light: '#C8C6C4', dark: '#333333', hc: '#000000' }, nls.localize('notebook.codeEditorToolbarBorder', "Notebook: Code editor toolbar right border"));
+export const codeCellParameterBackground = registerColor('notebook.codeCellParameterBackground', { light: '#0078D4', dark: '#0078D4', hc: '#0078D4' }, nls.localize('parameterTagBackground', "Parameter background color."));
```

Comment:
I think you can name this more generically, for example, notebookCellTagBackground

Reply:
A parameters tag can only be on a codeCell, but Im fine with making it more generic for the notebook level context.